config/grunt/custom-tasks/add-github-release.js

```diff
@@ -0,0 +1,139 @@
+const getUserInput = require( "../lib/get-user-input" );
+const githubApi = require( "../lib/github-api" );
+const uploadToGitHub = require( "../lib/upload-to-github" );
+
+/**
+ * Throws an error.
+ *
+ * @param {Object} response The response object.
+ * @param {Object} grunt    The grunt object.
+ *
+ * @returns {void}
+ */
+async function logError( response, grunt ) {
```

Comment:
The name of this function does not describe what it does accurately.

Suggesting to remove the `grunt.fail.fatal( "Failed to create a pre-release on GitHub." );` call out of the function.
This will make this function only log the error data. It does not have any knowledge of what response this is, so assuming it's a "pre-release" is not something it should do.

To: https://github.com/Yoast/wordpress-seo/pull/15910/files#diff-bad2168b62e65d8e4ddfbe9c6290e5a0R111
And: https://github.com/Yoast/wordpress-seo/pull/15910/files#diff-bad2168b62e65d8e4ddfbe9c6290e5a0R125

Reply:
The name of the function came from the original github-pre-release.js not sure what would be a more appropriate name: "HandleNotOkRespone"? 

changed the text to  `grunt.fail.fatal( "Failed to create a release on GitHub." );`